DepositeCalcTests/Tests/CalculatorPageTests.cs

```diff
@@ -0,0 +1,155 @@
+using DepositeCalcTests.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+
+namespace DepositeCalcTests.Tests
+{
+    internal class CalculatorPageTests
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void Setup()
+        {
+            ChromeOptions options = new ChromeOptions { AcceptInsecureCertificates = true };
+            driver = new ChromeDriver(options);
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(30);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5);
+            driver.Url = "https://localhost:5001/Calculator";
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
+
+        [TestCase("360", "1", "1", "")]
+        [TestCase("365", "1", "1", "")]
+        public void MandaroryTextFieldsTest(string financialYear, string depositAmount, string interestRate, string investmentTerm)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+
+            // Act
+            calculatorPage.FillingMandatoryTextFields(depositAmount, interestRate, investmentTerm);
+            calculatorPage.FinancialYear = financialYear;
+
+            // Assert
+            Assert.IsTrue(calculatorPage.IsCalculateBtnDisabled, "Calculate button is clickable. This means that one of the text fields or one of the Financial year radio buttons are not mandatory");
+        }
+
+        [TestCase("360", "100000", "100", "360", "200,000.00", "100,000.00")]
+        [TestCase("365", "100000", "100", "365", "200,000.00", "100,000.00")]
+        public void ValidCalculationTest(string financialYear, string depositAmount, string interestRate, string investmentTerm, string expectedIncome, string expectedInterestEarned)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+
+            // Act
+            calculatorPage.FinancialYear = financialYear;
+            calculatorPage.FillingMandatoryTextFields(depositAmount, interestRate, investmentTerm);
+            calculatorPage.Calculate();
+
+            // Asserts
+            Assert.AreEqual(expectedIncome, calculatorPage.Income, "Incorrect value in the Income field");
+            Assert.AreEqual(expectedInterestEarned, calculatorPage.InterestEarned, "Incorrect value in the Interest earned field");
+        }
+
+        [TestCase("360", "1000001", "10", "360")]
+        [TestCase("365", "1000000", "101", "365")]
+        [TestCase("360", "1000000", "100", "361")]
+        [TestCase("365", "1000000", "100", "366")]
+        public void NegativeCalculationTest(string financialYear, string depositAmount, string interestRate, string investmentTerm)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+
+            // Act
+            calculatorPage.FinancialYear = "365";
```

Comment:
I think it should be:
```suggestion
            calculatorPage.FinancialYear = financialYear;
```